AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOrchestrator.swift

```diff
@@ -85,26 +97,31 @@ final class AWSInitialSyncOrchestrator: InitialSyncOrchestrator {
 
     /// Enqueues sync operations for models and downstream dependencies
     private func enqueueSyncOperation(for modelType: Model.Type) {
-        let syncOperationCompletion: SyncOperationResultHandler = { result in
-            if case .failure(let dataStoreError) = result {
-                let syncError = DataStoreError.sync(
-                    "An error occurred syncing \(modelType.modelName)",
-                    "",
-                    dataStoreError)
-                self.syncErrors.append(syncError)
-            }
-        }
-
         let initialSyncForModel = InitialSyncOperation(modelType: modelType,
                                                        api: api,
                                                        reconciliationQueue: reconciliationQueue,
                                                        storageAdapter: storageAdapter,
-                                                       dataStoreConfiguration: dataStoreConfiguration,
-                                                       completion: syncOperationCompletion)
+                                                       dataStoreConfiguration: dataStoreConfiguration)
+
+        initialSyncOperationSinks[modelType.modelName] = initialSyncForModel
+            .publisher
+            .sink(receiveCompletion: { result in
```

Comment:
If you protect initialSyncOperationSinks with a queue, add `receive(on: queue)` here